GITPOD.md

```diff
@@ -0,0 +1,48 @@
+# Example Get Started Application in GitPod
+
+This repo holds an example Express.js application that uses FusionAuth as the identity provider. 
+This application will use an OAuth Authorization Code Grant workflow to log a user in and 
+get them access and refresh tokens.
+
+## Launch - GitPod
+
+The system is creating a fusionauth server, along with the backend containers needed to run it.  When it is done you will have a login for the system.
+
+Login to the server with 'admin@example.com' and password 'password' to verify that the installation has worked.  Once there, you can move on to the Delegate step.
+
+## Delegate Authentication
+
+The next step is to delegate authentication for your application, and verify that the change is applied.  For this you will:
```

Comment:
Are these instructions still correct? Do we need to keep them in sync with the instructions in fusionauth-site or can we just point people to the 'get started' section in the techdocs?

Reply:
Let's go over this when we review.  Because they get put *into* the environment, I'm a little wary of having them go out of the environment to read more instructions (because they might lose the tab or whatever).  I should be able to load the instructions from -site and maybe that's the right plan.

Reply:
So I think the right answer is to point them to an anchored part of the guide on -site, which we can open into a window in the environment.  I'll need to make sure the experience flows well.  I can't currently do this because I think the *right* thing is to point to a page on fusionauth.io, not GitHub, and the start-here isn't pushed yet.  Does that make sense?

Reply:
Makes sense. I get that this is a work in progress and that moving between two repos, one of which is not on main or published, makes things a bit tricky.